kafka-streams-framework/src/main/java/org/hypertrace/core/kafkastreams/framework/topics/creator/KafkaTopicCreator.java

```diff
@@ -0,0 +1,57 @@
+package org.hypertrace.core.kafkastreams.framework.topics.creator;
+
+import java.util.List;
+import java.util.Properties;
+import java.util.concurrent.ExecutionException;
+import java.util.stream.Collectors;
+import org.apache.kafka.clients.CommonClientConfigs;
+import org.apache.kafka.clients.admin.AdminClient;
+import org.apache.kafka.clients.admin.CreateTopicsResult;
+import org.apache.kafka.clients.admin.NewTopic;
+import org.apache.kafka.common.errors.TopicExistsException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Helper class for creating required topics before starting streaming application
+ */
+public class KafkaTopicCreator {
+  private static final Logger LOGGER = LoggerFactory.getLogger(KafkaTopicCreator.class);
+
+
+  private static final int numPartitions = 1;
```

Comment:
I and @laxmanchekka have discussed this. For now, we are going with hard-coded value. This can be made configurable if needed.